### PR TITLE
Pwa 555 redo toast messages

### PR DESCRIPTION
**Pull request by richardgorman**

### CODE REVIEW

File: pages/Favorites/subscriptions.js [hunk 1]
```diff
@@ -14,20 +14,21 @@ export default function favorites(subscribe) {
     dispatch(setTitle('titles.favorites'));
   });
 
-  subscribe(favoritesWillRemoveItem$, ({ dispatch, action, getState }) => {
+  subscribe(favoritesWillRemoveItem$, ({
+    action, dispatch, events, getState,
+  }) => {
     if (getCurrentRoute(getState()).pattern !== FAVORITES_PATH) {
```
Comment (devbucket) [inline pages/Favorites/subscriptions.js]: You should change this to use the router helper function instead.

File: pages/index.jsx [hunk 2]
```diff
@@ -15,6 +16,8 @@ import { CART_PATH } from '@shopgate/pwa-common-commerce/cart/constants';
 import { FAVORITES_PATH } from '@shopgate/pwa-common-commerce/favorites/constants';
 import { SEARCH_PATH } from '@shopgate/pwa-common-commerce/search/constants';
 import Portal from '@shopgate/pwa-common/components/Portal';
+import Toaster from '@shopgate/pwa-common/components/Toaster';
+import SnackBar2 from '@shopgate/pwa-ui-material/SnackBar';
```
Comment (devbucket) [inline pages/index.jsx]: I believe a mistake happened here , hm? ;)

File: pages/index.jsx [hunk 3]
```diff
@@ -37,38 +40,41 @@ const Pages = () => (
...
-            <Helmet>
-              <link href={devFontsUrl} rel="stylesheet" />
-            </Helmet>
-          )}
-        </Viewport>
+        <ToastProvider>
+          <Portal name={APP_GLOBALS} />
+          <Viewport>
+            <ModalContainer component={Dialog} />
+            <SnackBar />
```
Comment (devbucket) [inline pages/index.jsx]: What? Why? Still needed?

File: pages/index.jsx (hunk 2 above)
Comment (richardgorman) [reply pages/index.jsx]: Yeah, corrected

File: pages/index.jsx (hunk 3 above)
Comment (richardgorman) [reply pages/index.jsx]: No, removed

File: pages/Favorites/subscriptions.js (hunk 1 above)
Comment (richardgorman) [reply pages/Favorites/subscriptions.js]: Changed

**Pull request merged**